pages/common/httping.md

```diff
@@ -0,0 +1,27 @@
+# httping
+
+> Measures the latency of a webserver.
+
+- Measure latency of webserver using a URL:
+
+`httping -g {{URL}}`
+
+- Measure latency of webserver using a hostname:
+
+`httping -h {{hostname}}`
+
+- Measure latency using a specific port:
+
+`httping {{webserver}} -p {{port}}`
+
+- Measure latency only a specific number of times:
```

Comment:
Does this repeat the test multiple times (the default being 1) ? If so, I would suggest - "Repeat the test a given number of times"